parcel.go

```diff
@@ -13,48 +14,125 @@ func NewParcelStore(db *sql.DB) ParcelStore {
 }
 
 func (s ParcelStore) Add(p Parcel) (int, error) {
-	// реализуйте добавление строки в таблицу parcel, используйте данные из переменной p
-
-	// верните идентификатор последней добавленной записи
-	return 0, nil
+	// SQL-запрос для добавления новой записи в таблицу parcel
+	query := `INSERT INTO parcel (client, status, address, created_at) 
+	VALUES (:client, :status, :address, :created_at)`
+
+	// Выполнение запроса
+	result, err := s.db.Exec(query, sql.Named("client", p.Client), sql.Named("status", p.Status),
+		sql.Named("address", p.Address), sql.Named("created_at", p.CreatedAt))
+	if err != nil {
+		return 0, err
+	}
+
+	// Получение идентификатора добавленной записи
+	id, err := result.LastInsertId()
+	if err != nil {
+		return 0, err
+	}
+
+	return int(id), nil
 }
 
 func (s ParcelStore) Get(number int) (Parcel, error) {
-	// реализуйте чтение строки по заданному number
-	// здесь из таблицы должна вернуться только одна строка
-
-	// заполните объект Parcel данными из таблицы
-	p := Parcel{}
+	// SQL-запрос для получения одной записи по номеру посылки
+	query := `SELECT number, client, status, address, created_at FROM parcel WHERE number = :number`
+
+	// Выполнение запроса
+	row := s.db.QueryRow(query, sql.Named("number", number))
+
+	// Создание объекта для заполнения данными из БД
+	var p Parcel
+	err := row.Scan(&p.Number, &p.Client, &p.Status, &p.Address, &p.CreatedAt)
+	if err != nil {
+		if err == sql.ErrNoRows {
```

Comment:
Для сравнивания ошибок нужно использовать специализированную функцию
```suggestion
	require.ErrorIs(t, err, sql.ErrNoRows)
```